README.md

```diff
@@ -1 +1,44 @@
-# tdd_practice
\ No newline at end of file
+# OOP-School-Library
+
+## Built With
+
+- Ruby
+- Git
+
+## Getting Started
+
+To get started with this project, follow the below steps
+
+- Clone the repository using the following command `git clone git@github.com:DireAkanbi/OOP-School-Library.git`
+
```

Comment:
- [ ] It would be better to add some steps on how users can test your project locally.